pipelines/pipelines/nodes/search_engine/providers.py

```diff
@@ -239,3 +239,110 @@ def search(self, query: str, **kwargs) -> List[Document]:
         logger.debug("Serper.dev API returned %s documents for the query '%s'", len(documents), query)
         result_docs = documents[:top_k]
         return self.score_results(result_docs, len(answer_box) > 0)
+
+
+class SearchApi(SearchEngine):
```

Comment:
Thank you. Add a unitest for your SearchApi?

Reply:
Thanks for the review. Do you have any example test cases that use real HTTP requests? Otherwise, running the pipeline examples might be enough.